swap/src/bin/swap_cli.rs

```diff
@@ -112,14 +112,29 @@ async fn main() -> Result<()> {
             let monero_wallet =
                 init_monero_wallet(monero_network, monero_wallet_rpc_process.endpoint()).await?;
             let bitcoin_wallet = Arc::new(bitcoin_wallet);
+            let (event_loop, mut event_loop_handle) = EventLoop::new(
+                &seed.derive_libp2p_identity(),
+                alice_peer_id,
+                alice_addr,
+                bitcoin_wallet.clone(),
+            )?;
+            let handle = tokio::spawn(event_loop.run());
+
+            debug!("Requesting quote");
+
+            let bid_quote = event_loop_handle
+                .request_quote()
+                .await
+                .context("failed to request quote")?;
 
-            let swap_id = Uuid::new_v4();
+            info!("Received quote: 1 XMR = {}", bid_quote.price);
 
             // TODO: Also wait for more funds if balance < dust
             if bitcoin_wallet.balance().await? == Amount::ZERO {
                 info!(
-                    "Please deposit BTC to {}",
-                    bitcoin_wallet.new_address().await?
+                    "Please deposit BTC to {} (max {})",
```

Comment:
```suggestion
                    "Please deposit BTC (max {}) to be swapped to XMR to the swap-cli at address {}",
```

I think we need a bit more context here, so the user understands what will happen. It might also help to state that this BTC address is the swap-tool address. (Of course it is implied, but might not be bad to state it.)
Note that this would depend on the name of the application then - not super happy about that, but I think it helps the understanding.

Reply:
I don't think we need to explicitly state that, where this address comes from is an implementation detail IMO. People can look into the code and check whether it is actually an atomic swap :)